.github/workflows/docker-bench-security.yml

```diff
@@ -16,10 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
Consider renaming all these instances to "Set up Python 3" to eliminate future changes.

Reply:
done